2.6/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM php:7.2-cli-alpine3.7
+
+LABEL maintainer="frank.giesecke@final-gene.de"
+
+ENV PHPMD_VERSION 2.6.0
+
+WORKDIR /opt
+
+RUN apk add --no-cache --virtual=.build-deps bash
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN curl -LSs "https://github.com/phpmd/phpmd/archive/${PHPMD_VERSION}.tar.gz" | tar -xz
+
+WORKDIR /opt/phpmd-${PHPMD_VERSION}
```

Comment:
Would it not be safer to add "", to avoid/handle invalid overriding of the environment.